README.md

```diff
@@ -150,3 +150,93 @@ class WriteComment extends React.Component {
 
 export default connect(mapStateToProps, mapDispatchToProps)(WriteComment);
 ```
+
+### Cache Controlling
+```js
+import { schema } from 'normalizr';
+
+const BASE_URL = process.env.REACT_APP_API_BASE || '';
+const API_URL = BASE_URL;
+const cacheDuration = 60000;
+
+// define normalizr response schemas
+const searchTermSchema = new schema.Entity('SearchTerm');
+
+export default {
+    getSearchSuggestions: {
+        url: `${API_URL}/search/suggestions/?q=:searchTerm`,
+        method: 'GET',
+        cacheDuration,
+        searchTermSchema
+    }
+}
+```
+### After Success
+```js
+getCurrentUser: {
+    url: `${API_URL}/settings`,
+    method: 'GET',
+    afterSuccess: (request: any, dispatch: any, getState: any) => {
+        const data: IGetSettingsResponse = getResultData(getState().apiData, 'getSettings');
+        dispatch(update('app.currentUser', data.user.id));
+    }
+},
+```
+### Purge API Data (for deleting local storage)
+```js
+import { Dispatch } from 'redux';
+import { performApiRequest, purgeApiData } from 'react-api-data';
+
+export const logout = () => (dispatch: Dispatch) => {
+    dispatch(performApiRequest('postLogout'));
+    dispatch({ type: 'LOGOUT' });
+    dispatch(purgeApiData());
+};
+```
+
+
+### InvalidateRequest with Posting data
```

Comment:
Zou je deze en het voorbeeld hieronder (retrigger failed call) in het Posting Data voorbeeld kunnen verwerken?